src/Plugins/Parallel/Support/CompactPrinter.php

```diff
@@ -63,11 +63,19 @@ public function __construct(
      * Creates a new instance of the Compact Printer.
      */
     public static function default(): self
+    {
+        return self::create(true);
+    }
+
+    /**
+     * Creates a new instance of the Compact Printer with specific decoration setting.
+     */
+    public static function create(bool $decorated): self
     {
         return new self(
             terminal(),
-            new ConsoleOutput(decorated: true),
-            new Style(new ConsoleOutput(decorated: true)),
+            new ConsoleOutput(decorated: $decorated),
+            new Style(new ConsoleOutput(decorated: $decorated)),
```

Comment:
Added `create()` method to accept decoration parameter. Previously, CompactPrinter was hardcoded to use decorated output, ignoring `--colors=never` in parallel mode.